docs/narr/firstapp.rst

```diff
@@ -190,9 +190,8 @@ WSGI Application Creation
 
 After configuring views and ending configuration, the script creates a WSGI
 *application* via the :meth:`pyramid.config.Configurator.make_wsgi_app` method.
-A call to ``make_wsgi_app`` implies that all configuration is finished
-(meaning all method calls to the configurator, which sets up views and various
-other configuration settings, have been performed).  The ``make_wsgi_app``
+A call to ``make_wsgi_app`` implies that all configuration is finished (meaning all method calls to the configurator, which sets up views and various other configuration :term:`settings`, have been performed).
```

Comment:
this change doesn't make sense, the settings referred to here are not the literal `settings` object